WcaOnRails/app/models/registration.rb

```diff
@@ -171,6 +171,12 @@ def waiting_list_info
     OpenStruct.new(index: index, length: pending_registrations.length)
   end
 
+  def paid_waiting_list_info
+    pending_paid_registrations = competition.registrations.pending.order(registration_payments.map(&:created_at).max)
```

Comment:
I don't understand this. From what I can tell, `.order` is a method for retrieving data from the database using the `ORDER BY` SQL clause. Essentially, it sorts the data you're receiving.

You're passing in the **maximum** of the `created_at` dates, so it feels like "please order this list according to the 17th of January 1983" instead of saying "please order this list according to the creation date". What is your intention behind this line of code?